spec/expectations/grid_form_odk.xml

```diff
@@ -31,13 +39,18 @@
           </text>
         </translation>
       </itext>
+      <bind nodeset="/data/name2" readonly="true()" type="string"/>
```

Comment:
It seems like this binding is not used at all. Is it needed? (And the name2 tag above?)